perch/organisation.py

```diff
@@ -434,6 +433,9 @@ def authorized(self, requested_access, resource):
         :param resource: a Resource or SubResource with "permissions" attribute
         :returns: True if has access, False otherwise
         """
+        if {self.state, resource.state} != {State.approved.value}:
```

Comment:
We should also check the state of the parent - if the organisation is not approved then authorised should return false even if the service is approved